app/controllers/conferences_controller.rb

```diff
@@ -11,6 +11,11 @@ def index
 
   def show; end
 
+  def current
+    current = Conference.where('start_date <= ? AND end_date >= ?', Date.current, Date.current).first
```

Comment:
What if there is only 1 conference, which starts tomorrow?
We should redirect to the show action for this conference.